[CI] Failed to load driver class InvalidDriver from HikariConfig class classloader sun.misc.Launcher$AppClassLoader@42a57993

`commit id : e46fa62cc140c2728c9b024255759c47c25e4abe`

```
[INFO] Running org.apache.shardingsphere.infra.config.datasource.DataSourcePropertiesValidatorTest
[ERROR] 2022-01-12 15:35:50.633 [main] com.zaxxer.hikari.HikariConfig - Failed to load driver class InvalidDriver from HikariConfig class classloader sun.misc.Launcher$AppClassLoader@42a57993
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.111 s - in org.apache.shardingsphere.infra.config.datasource.DataSourcePropertiesValidatorTest
```

--- a/shardingsphere-infra/shardingsphere-infra-common/src/test/resources/logback-test.xml
+++ b/shardingsphere-infra/shardingsphere-infra-common/src/test/resources/logback-test.xml
@@ -25,7 +25,7 @@
     <logger name="org.apache.shardingsphere" level="warn" additivity="false">
         <appender-ref ref="console" />
     </logger>
-    <logger name="com.zaxxer.hikari" level="warn" />
+    <logger name="com.zaxxer.hikari" level="off" />
     
     <root>
         <level value="error" />

--- a/shardingsphere-jdbc/shardingsphere-jdbc-spring/shardingsphere-jdbc-core-spring/shardingsphere-jdbc-core-spring-namespace/src/test/resources/logback-test.xml
+++ b/shardingsphere-jdbc/shardingsphere-jdbc-spring/shardingsphere-jdbc-core-spring/shardingsphere-jdbc-core-spring-namespace/src/test/resources/logback-test.xml
@@ -25,6 +25,8 @@
     <logger name="org.apache.shardingsphere" level="warn" additivity="false">
         <appender-ref ref="console" />
     </logger>
+    <logger name="org.apache.shardingsphere.schedule.core.api.ModeScheduleContext" level="error" />
+    <logger name="org.apache.shardingsphere.mode.metadata.persist.service.ComputeNodePersistService" level="off" />
     
     <root>
         <level value="error" />

--- a/shardingsphere-mode/shardingsphere-mode-type/shardingsphere-cluster-mode/shardingsphere-cluster-mode-core/src/test/resources/logback-test.xml
+++ b/shardingsphere-mode/shardingsphere-mode-type/shardingsphere-cluster-mode/shardingsphere-cluster-mode-core/src/test/resources/logback-test.xml
@@ -25,6 +25,8 @@
     <logger name="org.apache.shardingsphere" level="warn" additivity="false">
         <appender-ref ref="console" />
     </logger>
+    <logger name="org.apache.shardingsphere.schedule.core.api.ModeScheduleContext" level="error" />
+    <logger name="org.apache.shardingsphere.mode.metadata.persist.service.ComputeNodePersistService" level="off" />
     
     <root>
         <level value="error" />

--- a/shardingsphere-mode/shardingsphere-mode-type/shardingsphere-standalone-mode/shardingsphere-standalone-mode-core/src/test/resources/logback-test.xml
+++ b/shardingsphere-mode/shardingsphere-mode-type/shardingsphere-standalone-mode/shardingsphere-standalone-mode-core/src/test/resources/logback-test.xml
@@ -1,0 +1,34 @@
+<?xml version="1.0"?>
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
+  ~ contributor license agreements.  See the NOTICE file distributed with
+  ~ this work for additional information regarding copyright ownership.
+  ~ The ASF licenses this file to You under the Apache License, Version 2.0
+  ~ (the "License"); you may not use this file except in compliance with
+  ~ the License.  You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<configuration>
+    <appender name="console" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>[%-5level] %d{yyyy-MM-dd HH:mm:ss.SSS} [%thread] %logger{36} - %msg%n</pattern>
+        </encoder>
+    </appender>
+    <logger name="org.apache.shardingsphere" level="warn" additivity="false">
+        <appender-ref ref="console" />
+    </logger>
+    <logger name="org.apache.shardingsphere.mode.metadata.persist.service.ComputeNodePersistService" level="off" />
+    
+    <root>
+        <level value="error" />
+        <appender-ref ref="console" />
+    </root>
+</configuration> 

--- a/shardingsphere-test/shardingsphere-integration-driver-test/src/test/resources/logback-test.xml
+++ b/shardingsphere-test/shardingsphere-integration-driver-test/src/test/resources/logback-test.xml
@@ -25,6 +25,8 @@
     <logger name="org.apache.shardingsphere" level="warn" additivity="false">
         <appender-ref ref="console" />
     </logger>
+    <logger name="org.apache.shardingsphere.schedule.core.api.ModeScheduleContext" level="error" />
+    <logger name="org.apache.shardingsphere.mode.metadata.persist.service.ComputeNodePersistService" level="off" />
     
     <root>
         <level value="error" />

